Makefile

```diff
@@ -23,8 +23,8 @@ run: build
 	  --region us-west-2 \
 		--cloudwatchregion us-west-1 \
 	  --workername `hostname` \
-	  --cmd ./bin/$(EXECUTABLE)
+	  --cmd ./bin/$(EXECUTABLE) -task=stale
```

Comment:
rather than including this here I would use the default property in the `configure` struct definition

Reply:
Sure, I can do that, though I'm always loathe to leave unclear defaults than then bite you later on.

Reply:
Yeah that makes sense but I’m this case having the default in the makefile would be confusing since you also have to make that same change in the Dockerfile for the default to exist when you deploy. 

Putting the default in the code means the behavior in local dev and docker is the same

Reply:
I didn't care about the default in the docker file - I'd actually *prefer* to have no default, and I was planning on updating the cronned tasks - as I mentioned in the PR notes.

The reason to update the local dev task (in my mind) was because I'd prefer to have ark start -l actually run and do something, rather than fail fast with an error because of a missing argument.
But if you'd be OK with no default and no argument for it in the Makefile, I'm fine with that, too.

Reply:
Current state seems most ideal to me